mobile/fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,5 +1,9 @@
-Cozy.io is a platform that brings all your web services in the same private space. With it, your web apps and your devices can share data easily, providing you with a new experience. You can install Cozy on your own hardware where no one profiles you.
+Cozy Cloud makes the personal cloud more accessible. Files and Photos you keep in your Cozy, your personnal cloud are accessible from everywhere.
```

Comment:
Need a comma.
"Files and Photos you keep in your Cozy, your personnal cloud, are accessible from everywhere."